.travis/publish-images.sh

```diff
@@ -25,4 +25,22 @@ if [ "${MAJOR_MINOR}x" != "x" ]; then
     MAJOR_MINOR_IMAGE="${BASE_BUILD_IMAGE}:${MAJOR_MINOR}"
     docker tag "${BUILD_IMAGE}" "${MAJOR_MINOR_IMAGE}"
     docker push "${MAJOR_MINOR_IMAGE}"
-fi
\ No newline at end of file
+fi
+
+## now, push to quay.io
+if [ "${QUAY_PASSWORD}x" != "x" -a "${QUAY_USERNAME}x" != "x" ]; then
```

Comment:
nice trick? 

Is it better than testing on empty env var?

Reply:
I guess I'm dating myself. There's probably a better (modern) way to do that, but old shells (in some Solaris versions, IIRC) didn't handle string comparisons correctly. I guess it's just one of those things that we get used to and never change :)

Reply:
@mpersano taught me that trick, perhaps he remembers what went wrong, where and why :)